warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -347,7 +383,11 @@ public int run(String[] args) throws Exception {
             job.addFileToClassPath(dependency);
         
         configureInputFormat(job, cbHelper, conf);
+        stop = System.currentTimeMillis();
+        log.trace("Time for setJarByClass=" + (stop - start));
```

Comment:
In addition to timing setJarByClass, this is adding files to the classpath and setting up the input format.